package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "govuk-react",
   "scripts": {
+    "yo": "yarn && yarn bootstrap && yarn build && echo WATCH && yarn watch",
```

Comment:
yarn will yarn bootstrap
yarn watch will yarn build

so you should just need `yarn && echo WATCH && yarn watch`